packages/jest/src/index.js

```diff
@@ -93,8 +93,6 @@ module.exports = neutrino => {
             [extensionsToNames(style)]: require.resolve('./style-mock')
           }),
       bail: true,
-      // eslint-disable-next-line camelcase
-      coveragePathIgnorePatterns: [node_modules],
```

Comment:
We don't need this line any more?

Reply:
The default is the regexp pattern `"/node_modules/"` which matches all node_modules:
https://facebook.github.io/jest/docs/en/configuration.html#coveragepathignorepatterns-array-string

The previous value was the custom `node_modules` setting which no longer exists.

Reply:
OK perfect.